packages/hagrid/hagrid/cli.py

```diff
@@ -2810,49 +2823,73 @@ def quickstart_setup(
+            if reset and os.path.exists(virtual_env_dir):
+                shutil.rmtree(virtual_env_dir)
+            env = VirtualEnvironment(virtual_env_dir, python=python)
+            console.print(
+                ":white_heavy_check_mark: Created Virtual Environment :evergreen_tree:"
+            )
 
-        if EDITABLE_MODE:
-            # local_syft_dir = Path(os.path.abspath(Path(hagrid_root()) / "../syft"))
-            # print("Installing Syft in Editable Mode")
-            # env.install("-e " + str(local_syft_dir))
-            local_hagrid_dir = Path(os.path.abspath(Path(hagrid_root()) / "../hagrid"))
-            print("Installing HAGrid in Editable Mode", str(local_hagrid_dir))
-            env.install("-e " + str(local_hagrid_dir))
-        else:
-            # options = []
-            # options.append("--force")
-            # if syft_version == "latest":
-            #     syft_version = LATEST_STABLE_SYFT
-            #     package = f"syft>={syft_version}"
-            #     if pre:
-            #         package = f"{package}.dev0"  # force pre release
-            # else:
-            #     package = f"syft=={syft_version}"
-
-            # if pre:
-            #     options.append("--pre")
-            #     print(f"Installing {package} --pre")
-            # else:
-            #     print(f"Installing {package}")
-            # env.install(package, options=options)
-            print("Installing hagrid")
-            env.install("hagrid", options=["-U"])
+            # upgrade pip
+            console_status.update("[bold blue]Installing pip :fish:")
+            env.install("pip", options=["-U"])
+            console.print(":white_heavy_check_mark: pip")
+
+            # upgrade packaging
+            console_status.update("[bold blue] Installing packaging")
+            env.install("packaging", options=["-U"])
+            console.print(":white_heavy_check_mark: packaging")
+
+            console_status.update("[bold blue]Installing Jupyter Labs")
+            env.install("jupyterlab")
+            env.install("ipywidgets")
+            console.print(":white_heavy_check_mark: Jupyter Labs")
+
+            if EDITABLE_MODE:
+                # local_syft_dir = Path(os.path.abspath(Path(hagrid_root()) / "../syft"))
+                # print("Installing Syft in Editable Mode")
+                # env.install("-e " + str(local_syft_dir))
+                local_hagrid_dir = Path(
+                    os.path.abspath(Path(hagrid_root()) / "../hagrid")
+                )
+                console_status.update(
+                    f"[bold blue]Installing HAGrid in Editable Mode: {str(local_hagrid_dir)}"
+                )
+                env.install("-e " + str(local_hagrid_dir))
+                console.print(
+                    f":white_heavy_check_mark: HAGrid in Editable Mode: {str(local_hagrid_dir)}"
+                )
+            else:
+                # options = []
```

Comment:
Q: should we remove the commented code?

Reply:
Not sure, its for installing syft. @madhavajay do we want to remove the syft installation code from quickstart or keep it for now ?

Reply:
Yeah you can remove that for now, basically until syft can install reliably without any deps on any system if we try at this stage it will fail. For example syft requires jax and jax isnt on pypi for windows, or previously with apple silicon requiring cmake to compile pycapnp. I suspect we will have these problems on certain systems for a while which is why we need a light weight setup tool to help install syft.

Reply:
Fixed via https://github.com/OpenMined/PySyft/pull/6770/commits/e447356ec45a72d4cf5cd161a920170832462b82